scapy/utils6.py

```diff
@@ -60,7 +61,7 @@ def cset_sort(x,y):
         cset = (x for x in laddr if x[1] == IPV6_ADDR_SITELOCAL)
     elif in6_ismaddr(addr):
         if in6_ismnladdr(addr):
-            cset = [('::1', 16, scapy.consts.LOOPBACK_INTERFACE)]
+            cset = [('::1', 16, LOOPBACK_INTERFACE)]
```

Comment:
Could you let the static import for this the LOOPBACK_INTERFACE, as it's updated by arch.windows